broader-topics/price-calculation-rules.md

```diff
@@ -83,3 +83,6 @@ This will ensure the addition available on the invoice is always correct no matt
 
 2. Specific rule - with round on each item type
```

Comment:
Hey @marionf, should we add the information about the round on each line to underline the difference between the two rounding types?